Shopping Site Js/main.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+from urllib.parse import urlparse
+import redis
```

Comment:
import modules should be in alphabetic order.
and import multiple modules in one line from the parent module.
like, `from werkzeug.routing import Map, Rule`